app/gateway/2.8.x/plan-and-deploy/security/secrets-management/getting-started.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Kong Secrets Management
+---
+
+{:.warning .no-icon}
+> **Beta warning:** 
+> In this version of the feature, you have to explicitly enable this feature. 
+> <br>
+> <br>
+> KONG_VAULTS=vault-name
+> <br>
+> ⚠️This feature is currently in beta state and isn't fully supported. APIs are subject to change.
```

Comment:
```suggestion
{:.warning .no-icon}
> **Beta warning:** 
> As this feature is in beta, APIs are subject to change. 
> In this version, you have to explicitly enable this feature by configuring the following parameter in your `kong.conf` file:
> ```
> KONG_VAULTS=vault-name
>```
```
There will already be a support warning before this paragraph with `beta: true`.